lib/prettier/2.0.x/.prettierrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
Also the [default value for `endOfLine` changed from `auto` to `lf`](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-endofline-to-lf-7435httpsgithubcomprettierprettierpull7435-by-kachkaevhttpsgithubcomkachkaev). 

Not sure whether we should use the default or set `auto` to keep the old behavior.

In fact we have this issue open related to this: #16 

I think we should test this very well before if we decide to switch to `lf`...

Reply:
Actually, you're right. We should keep `auto` as it is the default behavior for Git @NBB.
I'll set `auto` for configurations > 2.0.x. 

Thanks for your remark :+1: